ir/json_loader.h

```diff
@@ -223,10 +224,35 @@ class JSONLoader {
         load(::P4::get(obj, "value"), value), v = std::move(value);
     }
 
+    template <int N, class Variant>
+    std::enable_if_t<N == std::variant_size_v<Variant>, void> unpack_variant(
```

Comment:
Any reason for the explicit `void` in `enable_if_t` here and in the next overload?